api/computes_api.partials.yml

```
@@ -162,6 +162,124 @@
             schema:
               $ref: '#/components/schemas/ComputeSpec'
         description: Information about compute cluster
+      default:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Error'
+        description: unexpected error
+
+/computes/{computeId}/deployments:
+  ########################################
+  # Get deployments of a compute cluster
+  ########################################
+  get:
+    operationId: getComputeDeployments
+    summary: Get deployments for a compute cluster
+    tags:
+      - computes
+    parameters:
+      - name: computeId
+        description: compute id of compute cluster
+        in: path
+        schema:
+          type: string
+        style: simple
+        explode: false
+        required: true
+    responses:
+      "200":
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/ComputeJobStatus'
+        description: A paged array of status on job deployments in a compute cluster
+      default:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Error'
+        description: unexpected error
+
+/computes/{computeId}/deployments/{jobId}/config:
+  ########################################
+  # Get config for a job deployment on a compute cluster
+  ########################################
+  get:
+    operationId: getComputeJobConfig
```

How about the following?
```suggestion
    operationId: getDeploymentConfig
```